server/portal/templates/includes/nav_search.html

```diff
@@ -0,0 +1,24 @@
+{# @var className #}
+
+{# Encapsulate search bar as custom element, so styles do not bleed out #}
+{# IDEA: Should CMS use React component or Portal's React use custom element #}
+{# WARNING: Some markup is duplicated in other repositories #}
+{# SEE: https://confluence.tacc.utexas.edu/x/LoCnCQ #}
+<tacc-search-bar class="s-search-bar  {{className}}">
+  <!-- The search bar is populated here (from an external template). -->
+  {# FAQ: Content populated via JavaScript, below #}
+</tacc-search-bar>
+
+<script type="module">
+  import * as importHTML from '/static/site_cms/js/modules/importHTML.js';
+  import * as registerCustomElement from '/static/site_cms/js/modules/registerCustomElement.js';
+
+  importHTML.getFromURL('/cms/nav/search/markup/').then(markup => {
```

Comment:
Can we have the search bar populate with whatever the value of `query_string` is in the url?

Reply:
Yes. Task added.

Reply:
Done.

- On load, header search field populates with `query_string` value.
- On load, search page search field populates with `query_string` value.
- On Portal search page search, header search field updates with `query_string` value.

Reply:
Beautiful.